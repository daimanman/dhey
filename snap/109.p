{"C":1,"Duration":112698936,"EndTime":1551408724957772600,"Err":"","FileId":"","InputFileName":"","LonhApi":"signOnline","Method":"","N":1,"P":null,"Payload":"","QH":{},"Remark":"","ReqNums":1,"StartTime":1551408724845073700,"Status":1,"TaskId":109,"Type":"LONH","Url":"","Z":""}