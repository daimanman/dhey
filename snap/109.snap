{"AvgTotal":0.1066815,"Fastest":0.1066815,"Slowest":0.1066815,"Average":0.1066815,"Rps":9.373696470334595,"AvgConn":0.051829,"AvgDNS":0.0039975,"AvgReq":0,"AvgRes":0,"AvgDelay":0.0548525,"ConnMax":0.0039975,"ConnMin":0.0039975,"DnsMax":0.0039975,"DnsMin":0.0039975,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0,"DelayMax":0.0548525,"DelayMin":0.0548525,"Lats":[0.1066815],"ConnLats":[0.0039975],"DnsLats":[0.0039975],"ReqLats":[0],"ResLats":[0],"DelayLats":[0.0548525],"Total":106681500,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":0,"SizeReq":0,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.1066815,"Count":1,"Frequentcy":1},{"Mark":0.1066815,"Count":0,"Frequentcy":0},{"Mark":0.1066815,"Count":0,"Frequentcy":0},{"Mark":0.1066815,"Count":0,"Frequentcy":0},{"Mark":0.1066815,"Count":0,"Frequentcy":0},{"Mark":0.1066815,"Count":0,"Frequentcy":0},{"Mark":0.1066815,"Count":0,"Frequentcy":0},{"Mark":0.1066815,"Count":0,"Frequentcy":0},{"Mark":0.1066815,"Count":0,"Frequentcy":0},{"Mark":0.1066815,"Count":0,"Frequentcy":0},{"Mark":0.1066815,"Count":0,"Frequentcy":0}],"TaskId":109,"Duration":0.1066815}